topics/ecology/tutorials/MetaShARK_tutorial/tutorial.md

```diff
@@ -0,0 +1,309 @@
+---
+layout: tutorial_hands_on
+
+title: Creating metadata using Ecological Metadata Language (EML) standard
+  with EML Assembly Line functionalities
+zenodo_link: https://zenodo.org/records/10663465
+questions:
+- How to generate detailled metadata easily from biodiversity datasets ?
+- How to use international metadata standard?
+- How to update metadata informations ?
+objectives:
+- Explain the necessity of using such tools when producing ecological metadata
+- Learn how to use the interactive tool MetaShARK
+- Understand the challenges MetaShARK is trying to respond to
+- Learn how to create rich metadata using Ecological Metadata Language (EML) standard
+- Learn how to update EML metadata
+time_estimation: 30M
+key_points:
+- This tool aims to improve FAIR quality of metadata focusing on user exeprience and automatic inferences
+- Creating metadata as FAIR as possible is a must
+- Be carefull of the format and standard of metadata used only EML metadata will work
+tags:
+  - Metadata
+  - EML
+  - Ecology
+  - Biodiversity
+  - FAIR
+  - Data Paper
+contributions:
+    authorship:
+        - yvanlebras
+    editing:
+        - yvanlebras
+        - hexylena
+    funding:
+        - fnso2019
+        - pndb
+
+---
+
+
+<p align="justify">This tutorial aims to teach <b>how to use functionalities of the EML Assembly Line R package to produce rich metadata</b> using the Ecological Metadata Language (EML) international metadata standard. Here, we will notably propose a concrete example on how to use <b>Galaxy Ecology tools to create, evaluate and modify EML metadata</b> content using both EML Assemby Line metadata template tabular files, easily readable and editable by humans, and XML file, devoted to machine.</p>
+
+> <agenda-title></agenda-title>
+>
+> In this tutorial, we will cover:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+# 1] How can EML Assembly Line functionalities help producing rich metadata easily?
+
+<p align="justify"> A major gap when a researcher is writing metadata documents is the fact that metadata international <b>standards often use formats not really human readable</b> and/or editable as XML or JSON. To answer this issue, <b>Environmental Data Initiative</b> (insert:https://edirepository.org/) (EDI) through the <b>EML Assembly Line R package</b> propose to generate intermediate metadata template files using classical tabular text format.
```

Comment:
what is 'insert'? Is it supposed to be a link? You are using a lot of acronyms, do you want to use the acronym feature?

Reply:
hmm not sure.... I change it and delete the "insert" use